mypyc/emitmodule.py

```diff
@@ -721,20 +719,19 @@ def declare_finals(
             static_name = emitter.static_name(name, module)
             emitter.context.declarations[static_name] = HeaderDeclaration(
                 '{}{};'.format(emitter.ctype_spaced(typ), static_name),
+                [self.final_definition(module, name, typ, emitter)],
```

Comment:
I don't understand how this fixes the issue. It looks like `generate_c_for_modules` can still emit things twice, since there are two loops over declarations, both of which can emity stuff?